pulsar-broker/src/main/java/org/apache/pulsar/broker/service/persistent/PersistentTopic.java

```diff
@@ -1221,8 +1221,7 @@ private CompletableFuture<Void> delete(boolean failIfHasSubscriptions,
 
                         deleteTopicAuthenticationFuture.thenCompose(ignore -> deleteSchema())
                                 .thenCompose(ignore -> {
-                                    if (!this.getBrokerService().getPulsar().getBrokerService()
-                                            .isSystemTopic(TopicName.get(topic))) {
+                                    if (!SystemTopicNames.isTopicPoliciesSystemTopic(topic)) {
```

Comment:
topic policy system topic also can delete it topic policy, why we don't clean up the topic policy of the topic policy

Reply:
The topic policy topic will send a message to itself and then report `topic is fenced`.
And delete topic policy topic does not need to delete its policy.